lib/meow/topology.ex

```diff
@@ -73,7 +73,19 @@ defmodule Meow.Topology do
   """
   @spec fully_connected(number_of_populations(), population_index()) ::
           neighbour_population_indices()
-  def fully_connected(n, idx) do
+  def fully_connected(n, idx) when n > 1 do
     Enum.to_list(0..(n - 1)) -- [idx]
   end
+
+  @doc """
+  Represents a star topology.
+
+  The populations are arranged into a star, and the one
+  with `0` index resides in the middle of it.
+  """
+  @spec star(number_of_populations(), population_index()) ::
+          neighbour_population_indices()
+  def star(n, idx) when n > 1 and idx !== 0, do: [0]
+
+  def star(n, idx) when n > 1 and idx === 0, do: fully_connected(n, 0)
```

Comment:
We could use `if` or `case`, so that we don't need to repeat the `n > 1` guard, but your call :D

```suggestion
  def star(n, idx) when n > 1 do
    case idx do
      0 -> fully_connected(n, 0)
      _ -> [0]
    end
  end
```